### PR TITLE
React Frontend: add Flags and Labels

### PR DESCRIPTION
**Pull request by danielbeardsley**
Add flags on the pull cards for:

* `dev_block` signature
* `QAing` label
* `external_block` label

Note: Built on top of #246

### CODE REVIEW

**Comment by danielbeardsley** [comment]
QA :

image
image

**Comment by cdcline** [comment]
CR 👍

File: frontend/test/named-pulls.ts [hunk 1]
```diff
@@ -152,3 +152,20 @@ export const ManyStatuses = <PullData[]> [
       },
    }),
 ];
+
+const devBlock = sig(SignatureType.dev_block, true);
+const deployBlock = sig(SignatureType.deploy_block, true);
+export const Blocked = <PullData[]> [
```
Comment (mlahargou) [inline frontend/test/named-pulls.ts:159]: Maybe a pull with dev and deploy blocked so we make sure it's in the dev_blocked column?

**Comment by mlahargou** [comment]
CR 📱

File: frontend/test/named-pulls.ts (hunk 1 above)
Comment (danielbeardsley) [reply frontend/test/named-pulls.ts:159]: These are (at the moment) only used on the pull-card-demo page. I could probably find a way to use this as the main data-source for the dummy-pulls too, but that can happen in another pull.

**Comment by danielbeardsley** [comment]
Does you CRing here count for the pull this is based on? 

https://github.com/iFixit/pulldasher/pull/246

**Comment by mlahargou** [comment]
I didn't see that other pull b/c this is based off of master, not the other PR.

**Comment by deltuh-vee** [comment]
@danielbeardsley Some of the filled cards with more flags are cutoff
CleanShot 2022-03-16 at 14 51 57

**Comment by danielbeardsley** [comment]
> Some of the filled cards with more flags are cutoff

Hmm..  I should add some examples to the pull-card-demo page and then make those wrap.

**Comment by danielbeardsley** [comment]
> Hmm.. I should add some examples to the pull-card-demo page and then make those wrap.

Donezo!

image

QA :+1:

**Comment by cdcline** [comment]
CR 👍

**Pull request merged**